sklearn/metrics/regression.py

```diff
@@ -330,23 +330,38 @@ def mean_squared_log_error(y_true, y_pred,
                               sample_weight, multioutput)
 
 
-def median_absolute_error(y_true, y_pred):
+def median_absolute_error(y_true, y_pred, multioutput='uniform_average'):
     """Median absolute error regression loss
```

Comment:
Could you add a full stop.

Reply:
I am not sure which line you want me to add the full stop. If it is 334 (after the title, i.e. you want "Median absolute regression loss.") that would be incompatible with all other metrics in this file?

Reply:
OK so let it like this. Normally this is the standard with numpydoc

Reply:
OK, in that case all nitpicks have now been addressed.